pyccel/codegen/printing/ccode.py

```diff
@@ -1808,14 +1808,12 @@ def _print_Constant(self, expr):
         if expr == math_constants['inf']:
             self.add_import(c_imports['math'])
             return 'HUGE_VAL'
-        elif expr == math_constants['pi']:
+        if expr == math_constants['nan']:
             self.add_import(c_imports['math'])
-            return 'M_PI'
```

Comment:
Is there any reason why you removed the existing printing for values which do have an equivalent in C?

Reply:
i just tried to make it more general i did not give it a lot of thoughts, at the moment i thought that it might be better to hav the python values so its more consistent with python.